### DISCUSSION TITLE
Resistor Value for Pico & Midi Din

### DISCUSSION THREAD

**Discussion by NRKorsakov** [shanteacontrols/OpenDeck#260] [General]
Hi!

The resistor values in the midi schematics for the Picos and other 3V boards don't have a resistor value?

The schematic for the Teensy 2.0 (which I think is a 5V board?) does specify 220, but the other set of schematics fot the 3V boards omit the value.

Does anyone know the correct value?

Thanks!

![sch_midi_3v](https://github.com/shanteacontrols/OpenDeck/assets/25620049/255da0f3-bc14-4352-aa8c-8fc6737d5380)

**Comment by NRKorsakov** [comment]
I guess I see 220 for the input, but is the output the same?

**Reply by paradajz** [reply]
I don't understand which resistor are you reffering to - they all have values?

**Reply by paradajz** [reply]
Specifically for output - R17 is 33 ohms and R18 is 10 ohms.

**Comment by NRKorsakov** [comment]
Yes thank you I was looking for R17 and R18.

I was thinking those numbers were referring to something else.

Thanks again for your kind help.
